src/scripts/activities/activities.scss

```diff
@@ -0,0 +1,91 @@
+back-link {
+  width: 1160px;
+  margin: 0 auto;
+  display: block;
+}
+
+.activities {
+  width: 1200px;
```

Comment:
We should centralize a style for "wide" pages so when we have to do mobile we can change them all at once. In the meantime, maybe this should be `max-width`?